addons/csi/openstack/nodeplugin.yaml

```diff
@@ -31,6 +25,9 @@
 {{ if eq .Cluster.MajorMinorVersion "1.21" }}
 {{ $version = "v1.20.3 "}}
 {{ end }}
+{{ if eq .Cluster.MajorMinorVersion "1.22" }}
+{{ $version = "v1.20.3 "}}
+{{ end }}
```

Comment:
This should match the version used in `addons/csi/openstack/controllerplugin.yaml`.